checking grammar on operations and ops accessing cluster page, updati… (#3075)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md

The URL provided in these details offers access to the **operations portal**, a web UI which links to various dashboards of the tooling integrated on Konvoy Kubernetes clusters.
This UI is hosted on a web application within the cluster, which runs on the cluster's ingress load balancer.

To access the UI:

1. Enter that URL in your browser.
2. Click the "Launch Console" button
3. Enter your username and password as noted above, and click "Login".

> You will then see Konvoy's operations portal, which offers an overview of cluster status, and shortcuts to several dashboards to addon services, such as Grafana.

Reviewer: ```suggestion
You will the Konvoy's operations portal. The portal offers an overview of cluster status, and shortcuts to several dashboards to addon services, such as Grafana.
```

Author: I think it's more clear the other way for this one

> <p class="message--important"><strong>IMPORTANT: </strong>When using an automated provisioner, you should not delete the load balancer created by the cloud provider for the traefik service. If the load balancer is deleted, you can follow the steps below to regain access to the ops-portal. In order to complete the below instructions, you need to update your kubectl credentials with the `admin.conf` file created for your Konvoy cluster by running the `./konvoy apply kubeconfig` command. More on that <a href="#configure-kubectl-for-cluster-administrators">here.</a></p>

Reviewer: ```suggestion
<p class="message--important"><strong>IMPORTANT: </strong>When using an automated provisioner, do not  delete the load balancer created by the cloud provider for the traefik service. If the load balancer is deleted, follow the steps below to regain access to the ops-portal. In order to complete these instructions, you must update your kubectl credentials with the <code>admin.conf</code> file created for your Konvoy cluster by running the <code>./konvoy apply kubeconfig</code> command. More on that <a href="#configure-kubectl-for-cluster-administrators">here.</a></p>
```

- Manage cluster resources
- View logs and status messages

For a complete list of `kubectl` operations, see [overview of kubectl][kubectl_overview].

### Install kubectl

> The steps required to install kubectl depends on your operating system platform.

Author: ```suggestion
The steps required to install kubectl depend on your operating system platform.
```

> The specific steps required in installing kubectl depend on your operating system platform.

Reviewer: ```suggestion
The steps required to install kubectl depends on your operating system platform.
```
